### DISCUSSION TITLE
How to use `tauri::api::dir::read_dir` from v1 in v2 in rust side?

### DISCUSSION THREAD

**Discussion by dmytro587** [tauri-apps/tauri#11868] [Q&A] [answered]
How to use `tauri::api::dir::read_dir` from v1 in v2 in rust side?

I see that `read_dir` becomes `[tauri::comand]` in fs plugin, but how to use it on the Rust side?

**Comment by FabianLars** [comment, marked as answer]
Use https://doc.rust-lang.org/std/fs/fn.read_dir.html instead
